### PR TITLE
Web 146 accessibility

### PR DESCRIPTION
**Pull request by ana-buildt**
- Add tests for the following pages: homepage, about, careers, locations.
- Add commands for the following features: scroll reveal, smooth scroll, toggle menu.

### CODE REVIEW

**Comment by ThePeach** [comment]
@franz899 agreed on the change of names for the scripts

File: docs/tests.md [hunk 1]
```diff
@@ -6,6 +6,7 @@
 * Live run the test suite with `npm run cypress:open`
 * Run the tests in a headless environment with `npm run cypress:cli`
 * Travis will invoke `npm run cypress:travis` to run the tests
+* If you're having issues with port 8080 being already in use run this command: `kill -9 $(lsof -t -i:8080)`
```
Comment (ThePeach) [inline docs/tests.md]: these instructions need to say that the webserver that cypress runs needs to be done over a build. That OR include the `npm run build` to the cypress script.

same goes for the accessibility tests.

File: docs/tests.md [hunk 2]
```diff
@@ -1,20 +1,26 @@
...
 * Learn more about [Cypress](https://www.cypress.io/support/)
 
 
-## Accessibility testing with Pa11y CI
-* Build the website locally and run the tests with `npm run accessibility`
-* Update test runner setup at [.pa11yci.json](../.pa11yci.json)
-* Learn more about [Pa11y CI](https://github.com/pa11y/pa11y-ci)
+# Accessibility testing with Pa11y CI
+## Run Locally
+* Run the website locally with `npm run start`
```
Comment (ThePeach) [inline docs/tests.md]: AFAIK you just need to run `npm run build`, please double check.

Comment (ana-buildt) [reply docs/tests.md]: It runs against the local server.

**Pull request merged**